### PR TITLE
add compare_images in no proxy list

### PR DESCRIPTION
**Pull request by KazuCocoa**
~~do not merge yet~~

https://discuss.appium.io/t/compare-images-with-appium-and-opencv-using-windowsdriver/25469

### CODE REVIEW

**Comment by KazuCocoa** [comment]
https://github.com/Microsoft/WinAppDriver/issues/630

**Comment by mykola-mokhnach** [comment]
I think it makes sense to rather create such PR to the original repository https://github.com/jonstoneman/appium-windows-driver

**Comment by KazuCocoa** [comment]
I got an answer about the repo.

https://github.com/jonstoneman/appium-windows-driver/pull/8#issuecomment-479017203

> this repo has become stale

----

Will wait for Microsoft/WinAppDriver#630 ' reaction, anyway

**Comment by KazuCocoa** [comment]
https://github.com/Microsoft/WinAppDriver/issues/630#issuecomment-479044913

**Comment by yodurr** [comment]
This PR looks good to me, and seems like a good addition while compare_images is not supported by WinAppDriver

**Comment by KazuCocoa** [comment]
ah, I see (I misunderstood..).

thanks

**Comment by yodurr** [comment]
Also this is the main repository for appium-windows-driver.  The jonstoneman one was the original back when we started integrating WinAppDriver with Appium, but then the project became part of the Appium family - so this is the correct place to submit such a PR.

File: lib/driver.js [hunk 1]
```diff
@@ -77,6 +84,11 @@ class WindowsDriver extends BaseDriver {
     return true;
   }
 
+  getProxyAvoidList (sessionId) {
+    super.getProxyAvoidList(sessionId);
```
Comment (imurchie) [inline lib/driver.js]: If we're not using the result from this, why make this call?

**Comment by KazuCocoa** [comment]
Can I merge this? 👀

**Comment by KazuCocoa** [comment]
thank you!

**Pull request merged**